acceptance/tests/custom_facts/using_win32ole_should_not_hang.rb

```diff
@@ -0,0 +1,29 @@
+test_name 'C92060: Custom facts should not hang Facter when using win32ole' do

```

Comment:
See the following github article for correcting the line ending issue.
https://help.github.com/articles/dealing-with-line-endings/